prime-router/src/main/kotlin/TransportType.kt

```diff
@@ -23,7 +23,8 @@ data class SFTPTransportType
 @JsonCreator constructor(
     val host: String,
     val port: String,
-    val filePath: String
+    val filePath: String,
+    val credentials: String? = null
```

Comment:
I think credentialName is a better variable name.   This string is not the actual credentials, its just the name used for the credentials in the vault.

Reply:
fixed